hivemq4/base-image/Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM openjdk:11-jre-slim
+FROM openjdk:11-jre-slim as javabase
 
-ARG HIVEMQ_VERSION=4.0.0
 ENV HIVEMQ_GID=10000
 ENV HIVEMQ_UID=10000
```

Comment:
These should be ARGs as they have no effect when set later for the containers.

Reply:
the ARGs should also be moved to the right build step

Reply:
These need to be ENVs, because they are used in the entrypoint scripts, but they need to be moved to he last step.